browser-test/bin/fmt

```diff
@@ -4,8 +4,10 @@ pushd $(git rev-parse --show-toplevel)/browser-test
 
 CMD="cd /code; npx prettier --write --config /.prettierrc.js --ignore-path /.prettierignore /code"
 
+bin/pull-image
+
 docker run --rm -it \
   -v "$(pwd)/src:/code" \
-  docker.io/civiform/formatter:latest sh -c "$CMD"
+  civiform/formatter:latest sh -c "$CMD"
```

Comment:
likewise here. why is docker.io not good?

Reply:
It's the default, IMO it made more sense to include it when we had shell scripts that pushed to both docker hub and AWS ECR just to have extra clarity. Now that everything in the main repo is only interacting with docker hub it's inconsistent to have it a few places and less needed for disambiguation.

Reply:
This is just removing extra specificity that we don't need, and making it act like our other scripts.  The "docker.io" is a repository name,  so it just tells the command to go to docker.io to get the image if it's not local.  But since it's also the default repository, docker already knows to look there.

Reply:
Also, since we are adding it to the pull-image step, it's already downloaded